src/clients/VKontakte.php

```diff
@@ -76,15 +76,15 @@ class VKontakte extends OAuth2
      * @see https://vk.com/dev/versions
      * @since 2.1.4
      */
-    public $apiVersion = '3.0';
+    public $apiVersion = '5.0';
 
 
     /**
      * {@inheritdoc}
      */
     protected function initUserAttributes()
     {
-        $response = $this->api('users.get.json', 'GET', [
+        $response = $this->api('users.get', 'GET', [
```

Comment:
Does calling for a `json` no longer work on vk? I know it shouldn't matter regardless of using it or not, but why change it if it still works?

Reply:
`.json` isn't part of the endpoint, it's a format you can add to any endpoint. For example, it could be `users.get.xml` so I don't think it should be changed.